lib/wanda_web/schemas/accepted_execution_response.ex

```diff
@@ -0,0 +1,23 @@
+defmodule WandaWeb.Schemas.AcceptedExecutionResponse do
+  @moduledoc """
+  Minimal information about an Execution accepted by the system,
+  it carries the same identifiers provided by the consumer that triggered the request.
+
+  These identifiers may be used to query the APIs about the state of an execution.
+  """
+
+  require OpenApiSpex
+
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "AcceptedExecution",
+    description:
+      "Identifiers of the Accepted Execution, status needs to be checks at appropriate API",
```

Comment:
This text looks wrong hehe
Why not something simpler like:
```
Execution and Group id of the recently started execution
```

Reply:
Changing to `Identifiers of the recently accepted execution`.

> What does status needs to be checks at appropriate API mean?

It means that in order to get the actual information about the execution, the appropriate API (get execution detail) needs to be called.

But that's not important anymore :smile: